models.py

```diff
@@ -0,0 +1,6 @@
+from keras.optimizers import Adam
+from keras.metrics import accuracy, binary_crossentropy, categorical_crossentropy
+from segmentation_models import Unet
+from segmentation_models.metrics import iou_score
+from segmentation_models.losses import jaccard_loss, dice_loss
+
```

Comment:
So this file is entirely new? Where does `generate_compiled_segmentation_model` show up since you're importing it in `train_seg_model.py`?

Reply:
yep it's new -- the idea is that previously unet (or whatever model/parameters we wanted to use) was hard coded into train, test, and infer. this makes it a real pain to try out different parameters and architectures. by introducing models.py we can make any model changes there (or really in the config file) and then have those used in train, test, and infer without having to touch their code